main.cpp

```diff
@@ -1,2 +1,6 @@
 int main() {
+  int a, b;
```

Comment:
warning: multiple declarations in a single statement reduces readability [readability-isolate-declaration]

```suggestion
  int a;
  int b;
```